deye_sensors.py

```diff
@@ -47,39 +47,48 @@
 pv1_current_sensor = SingleRegisterSensor("PV1 Current", 0x6e, 0.1, mqtt_topic_suffix='dc/pv1/current')
 pv1_power_sensor = ComputedPowerSensor("PV1 Power", pv1_voltage_sensor,
                                        pv1_current_sensor, mqtt_topic_suffix='dc/pv1/power')
+pv1_daily_sensor = SingleRegisterSensor("PV1 Daily", 0x41, 0.1, mqtt_topic_suffix='dc/pv1/daily')
```

Comment:
@Antagor To keep mqtt topic names consistent with already existing entries, I'd like you to rename all newly added `/daily` topics to `/day_energy`. Thanks

Reply:
done